c++/src/TypeImpl.cc

```diff
@@ -609,6 +614,8 @@ namespace orc {
       return std::unique_ptr<Type>(new TypeImpl(BINARY));
     } else if (category == "timestamp") {
       return std::unique_ptr<Type>(new TypeImpl(TIMESTAMP));
+    } else if (category == "timestamp_with_local_time_zone") {
```

Comment:
To be consistent with the Java side, the type name literal should remove the underscores: [commit](https://github.com/omalley/orc/blob/8289f7d2640833bba46be88941d1013eb2083023/java/core/src/java/org/apache/orc/TypeDescription.java#L124)

Reply:
Done, the type name has been modified to be consistent with the Java side.